code-runner.yaml

```diff
@@ -1,19 +1,18 @@
-stages:
-  serverConfig:
-    - from:
-        image: golang:latest
-    - env:
-        GOMODULE: on
-        GOFLAGS: -mod=vendor
-    - copy:
-        basedir: .
-        destdir: /code-runner
-    - workdir:
-        dir: /code-runner
-    - runCommand:
-        param: go build -o code-runner-server
-    - serverPort:
-        number: 8082
-    - cmd:
-        params:
-          - ./code-runner-server
\ No newline at end of file
+serverConfig:
+  - from:
+      image: golang:latest
+  - env:
+      GOMODULE: on
+      GOFLAGS: -mod=vendor
+  - copy:
+      basedir: .
+      destdir: /code-runner
+  - workdir:
+      dir: /code-runner
+  - runCommand:
+      param: go build -o code-runner-server
+  - serverPort:
```

Comment:
change to `port` as we already know that it's for the server.

Reply:
Done